### ISSUE TITLE
Do we want an "abandoned" request state?

### ISSUE THREAD

**Issue by baltpeter** [datenanfragen/website#984]
I'm currently going through my old requests (mostly in an effort to get rid of the red badge—it's very effective :D).

I noticed that I have a couple of requests that the respective company never properly responded to but that I stopped caring about along the way (yep, sometimes those tactics work on me as well :/). I don't want to mark them as completed (since they aren't) but I don't want to delete them either. Currently, there is no way for me to get rid of their overdue status under those constraints.

I thus propose adding an "abandoned" request state for this use case that doesn't bug the user about the request anymore but still allows them to react should they so choose in the future.

I'm not particularly fond of the risk of encouraging users to stop fighting for their rights but realistically that is sometimes going to happen anyway. And if we keep the notification in those cases, that just distracts from other overdue requests that the user may actually want to follow up on.

**Comment by mal-tee** [comment]
Maybe we should hide that behind a user setting? "Enable 'abandoned' state" or sth like that?

**Comment by zner0L** [comment]
I agree this would be a useful state. I think we shouldn't have a quick access button for that, though. But maybe an option to do "mass abandoning" in the select menu on the "my requests" page.
